README.md

```
@@ -293,6 +293,154 @@ Copied secrets are named `<fjob-name>-<secret-name>` and carry
+## PSAPCluster — Cluster Management
+
+The `PSAPCluster` custom resource provides a single pane of glass for cluster
+state, GPU inventory, and ownership locking.
+
+### Viewing clusters
+
+```bash
+oc get psapclusters -n psap-automation
+```
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)